src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -658,13 +651,13 @@ namespace OpenMS
         continue;
       }
        
-      countPeptides_(f.getPeptideIdentifications(), 1);
+      countPeptides_(f.getPeptideIdentifications());
       PeptideHit hit = getAnnotation_(f.getPeptideIdentifications());
       FeatureHandle handle(0, f);
       const size_t fraction(1), sample(1);
       quantifyFeature_(handle, fraction, sample, hit); // updates "stats_.quant_features"
     }
-    countPeptides_(features.getUnassignedPeptideIdentifications(), 1);
```

Comment:
Would it make sense to annotate peptide counts (spectral counts) by default as meta value? e.g. by running quantification twice…

Reply:
Yes.